src/components/Layout/Feedback.tsx

```diff
@@ -60,8 +60,27 @@ function sendGAEvent(isPositive: boolean) {
 
 function SendFeedback({onSubmit}: {onSubmit: () => void}) {
   const [isSubmitted, setIsSubmitted] = useState(false);
+  const [isVisible, setIsVisible] = useState(true);
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout | null = null;
+
+    if (isSubmitted) {
+      timer = setTimeout(() => {
+        setIsVisible(false);
+      }, 3000);
+    }
+
+    return () => {
+      if (timer) {
+        clearTimeout(timer);
+      }
+    };
+  }, [isSubmitted]);
   return (
-    <div className="max-w-xs w-80 lg:w-auto py-3 shadow-lg rounded-lg m-4 bg-wash dark:bg-gray-95 px-4 flex">
+    <div
+      className="max-w-xs w-80 lg:w-auto py-3 shadow-lg rounded-lg m-4 bg-wash dark:bg-gray-95 px-4 flex"
+      style={{opacity: isVisible ? 1 : 0}}>
```

Comment:
It would be nice if it come out with fade translation. 
Add `transition-opacity` tailwind class.

Reply:
Done with 300ms